internal/commands/setup.go

```diff
@@ -233,6 +262,14 @@ func doSetup(root *cobra.Command, cf configFactory, overrideCP clientProvider) {
 		// directory before we change it
 		var envFiles []string
 		if envFile != "" {
+			// handle http(s) location
+			if strings.HasPrefix(envFile, "http") {
```

Comment:
I think we should check for the schema like "http://" or 'https://' as there could be a file named httptestenv for example